### ISSUE TITLE
Custom float labels don't appear when step value is greater than 1

### ISSUE THREAD

**Issue by shahvaiz** [simeydotme/jQuery-ui-Slider-Pips#49]
I am trying to add custom float labels, but when I make the step value equal to 20, the custom floats don't appear. If I make the step equal to 1, then the custom floats do appear correctly.

https://jsfiddle.net/shanif/5x81Lzjq/

```
$(function() {
    $( "#slider" ).slider({
      min: 580,
      max: 800,
      step: 20,
    })
    .slider("float",  {
        labels: ["a", "b", "c", "d", "e", "f", "g", "h", "i", "j", "k", "l"]
    })
    .slider("pips") 
  });
```

**Comment by simeydotme** [comment]
Thanks for raising this @shahvaiz  !! :)
I will investigate asap!!

**Comment by simeydotme** [comment]
https://github.com/simeydotme/jQuery-ui-Slider-Pips/releases/tag/v1.7.6

Your issue should be resolved with version v1.7.6 :smile: - sorry about the bug!!
